### PR TITLE
IE Edge fixes

### PR DESCRIPTION
**Pull request by wfortin**
! Use fixed header + margin and remove vh units in calc since they are not supported in IE

### CODE REVIEW

File: scss/components/header.scss [hunk 1]
```diff
@@ -1,5 +1,9 @@
 .header {
+  position: fixed;
+  z-index: 1000;
+
   display: flex;
+  width: 100%;
   height: $header-height;
   padding-bottom: $header-application-offset-y;
```
Comment (marilynechan) [inline scss/components/header.scss]: If there's no offset anymore shouldn't this disappear?

File: scss/components/header.scss [hunk 2]
```diff
@@ -1,5 +1,9 @@
 .header {
+  position: fixed;
+  z-index: 1000;
```
Comment (marilynechan) [inline scss/components/header.scss]: Why a sooo big z-index :-( .

File: scss/components/application-container.scss [hunk 3]
```diff
@@ -1,8 +1,7 @@
 .application-container {
-  height: calc(100vh - #{$header-height});
-  margin-top: -1 * $header-application-offset-y;
+  margin-top: $header-height;
   margin-left: $navigation-width;
-  overflow-y: scroll;
+  overflow: hidden;
```
Comment (marilynechan) [inline scss/components/application-container.scss]: Auto?

**Comment by marilynechan** [comment]
Ship it!

File: scss/components/application-container.scss (hunk 3 above)
Comment (wfortin) [inline scss/components/application-container.scss]: No the scroll is taken care of by the body.

File: scss/components/header.scss (hunk 2 above)
Comment (wfortin) [inline scss/components/header.scss]: well I actualy didn't know which value to use. I guess i can go to 50.

File: scss/components/header.scss (hunk 1 above)
Comment (wfortin) [inline scss/components/header.scss]: Removing this.

**Comment by pochretien** [comment]
:ship:  it!

**Pull request merged**